lib/agent.js

```diff
@@ -86,7 +86,7 @@ Agent.prototype.clearPatches = function (modules) {
   return this._instrumentation.clearPatches.apply(this._instrumentation, arguments)
 }
 
-Agent.prototype.startTransaction = function (name, type, { startTime, childOf } = {}) {
+Agent.prototype.startTransaction = function (name, type, subtype, action, { startTime, childOf } = {}) {
```

Comment:
Remember to update the TypeScript typings as well

Reply:
fixed